src/e2e-test/resources/pluginParameters.properties

```diff
@@ -229,3 +229,19 @@ gcsMoveReadRecursivePath=testdata
 gcsMoveReadRecursiveSubDirectory=testdata/GCS_RECURSIVE_TEST
 locationEU=eu
 ## GCSMove-PLUGIN-PROPERTIES-END
+
+## BQArgumentSetter-PLUGIN-PROPERTIES-START
+bqArgumentSetterInvalidColumn=testInput
+bqArgumentSetterBlankArgSelectionConditionValue=[{"key":"fhddv","value":""}]
+bqArgumentSetterConditionName=[{"key":"name","value":"iqbal"}]
+bqArgumentSetterConditionName2=[{"key":"name","value":"iqbal"},{"key":"age","value":"20"}]
+bqArgumentSetterConditionInvalidName=[{"key":"name","value":"iqbal"},{"key":"name","value":"test_automation"}]
+bqArgumentSetterColumns=dataset
+bqArgumentSetterReferenceName=Test
+bqArgumentSetterReferenceNameSink=test1
+bqArgumentSetterDataSetSink=dataset
+bqArgumentCreateTableQueryFile=testdata/BigQuery/BigQueryArgumentCreateTableQuery.txt
```

Comment:
Add `bqArgumentSetter` prefix to both(create, insert) query file properties
Also add it to filename BigQueryArgumentCreateTableQuery.txt -> BigQueryArgumentSetterCreateTableQuery.txt

And Remove 2 unused query file properties (bqArgumentSetCreateTableQueryFile and bqArgumentSetInsertDataQueryFile), also remove unused txt query files.